node/config/types.go

```diff
@@ -426,6 +426,13 @@ type SealingConfig struct {
 	TerminateBatchMin  uint64
 	TerminateBatchWait Duration
 
+	// When submitting several sector prove commits message simultaneously, this option allows you to
+	// stagger the number of prove commits submitted per epoch
+	// This is done because gas estimates for ProveCommits are non deterministic and increasing as a large
+	// number of sectors get committed within the same epoch resulting in occasionally failed msgs.
+	// Submitting a smaller number of prove commits per epoch would reduce the possibility of failed msgs
+	MaxSectorProveCommitsSubmittedPerEpoch uint64
```

Comment:
Can we move this above `Terminate*` stuff, so all commit things are in one block?